core/types/trace.go

```diff
@@ -18,6 +20,17 @@ type ContractCodeUsage struct {
 	Write HexBytes        `json:"write,omitempty"`
 }
 
+type Uint256 struct {
+	uint256.Int
+}
```

Comment:
Can it be the type alias and it is enough overriding the `MarshalText` function. WDYT?

```suggestion
type Uint256 uint256.Int
```